File: Final 2018-3/srcgen/Empleado.rb
	
class Empleado

	def initialize(this,nombre,apellido,identificacion,area,sueldo)
		@nombre = nombre
		@apellido = apellido
		@identificacion = identificacion
		@area = area
		@sueldo = sueldo
	end
	
	def nombre
		return @nombre
	end

	def apellido
		return @apellido
	end

	def identificacion
		return @identificacion
	end

	def area
		return @area
	end

	def sueldo
		return @sueldo
	end

    def nombre=(nombre)
		@nombre = nombre
    end
	
    def apellido=(apellido)
		@apellido = apellido
    end
	
    def identificacion=(identificacion)
		@identificacion = identificacion
    end
	
    def area=(area)
		@area = area
    end
	
    def sueldo=(sueldo)
		@sueldo = sueldo
    end
	
end

File: Final 2018-3/srcgen/Empleado.cpp
class Empleado{

  Empleado(this){
  }
  Empleado((this,nombre,apellido,identificacion,area,sueldo):
      this.nombre=nombre
      this.apellido=apellido
      this.identificacion=identificacion
      this.area=area
      this.sueldo=sueldo

  void getNombre(){
    return this.nombre
	}
  void getApellido(){
    return this.apellido
	}
  void getIdentificacion(){
    return this.identificacion
	}
  void getArea(){
    return this.area
	}
  void getSueldo(){
    return this.sueldo
	}
  String setNombre(this,nombre){
    this.nombre = nombre
  }

  String setApellido(this,apellido){
    this.apellido = apellido
  }

  String setIdentificacion(this,identificacion){
    this.identificacion = identificacion
  }

  String setArea(this,area){
    this.area = area
  }

  String setSueldo(this,sueldo){
    this.sueldo = sueldo
  }

};